openstack/compute/v2/extensions/resetstate/requests.go

```diff
@@ -0,0 +1,23 @@
+package resetstate
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// ResetStatus refers to the states usable in ResetState Action
+type ResetStatus string
```

Comment:
I apologize - I didn't catch this before. I think having `Status` and `State` might be confusing. We can't use `ResetState` because that's the name of the function.

Using `State` by itself is ambiguous (the state of _what_?) So in that case, how about instead of `ResetStatus`, use `ServerState`. This removes the word "status" and gives more detail to what the status is referring to.

`StateActive` and `StateError` are both correct - just their type will now be `ServerState`.